httpcore/_async/connection_pool.py

```diff
@@ -76,6 +76,9 @@ class AsyncConnectionPool(AsyncHTTPTransport):
     * **keepalive_expiry** - `Optional[float]` - The maximum time to allow
     before closing a keep-alive connection.
     * **http2** - `bool` - Enable HTTP/2 support.
+    * **family** - `Optional[int]` - Address family to use
```

Comment:
Nit:

```suggestion
    * **family** - `int` - Address family to use, defaults to 0.
```

Reply:
I'll fix this.

Reply:
Shouldn't the type of the `family` parameter be `socket.AddressFamily` rather than just `int`?

Reply:
If all of occurrences of `0` were replaced with `socket.AF_UNSPEC`, then sure.  I don't want to make this change yet, though, since if there's consensus to combine family and address into a single field, then there wouldn't be a need for either one, because the entire field would be `None`.